spec/javascripts/live_search_spec.js

```diff
@@ -190,6 +192,15 @@ describe("liveSearch", function(){
       expect(liveSearch.updateResults).toHaveBeenCalled();
     });
 
+    it("should update email and atom subscription links when checkbox is changed", function(){
+      spyOn(liveSearch, 'updateLinks');
+      $form.find('input[name="field"]').prop('checked', false);
+
+      liveSearch.formChange();
+      expect(liveSearch.state).toEqual([{name: 'published_at', value: '2004'}]);
+      expect(liveSearch.updateLinks).toHaveBeenCalled();
```

Comment:
Presumably you're not explicitly checking the new href attributes on the links because you don't know what they'd be? If that's the case we could at least test that they are no longer equal to their initially set values.

Reply:
added new unit test that should provide more certainty